plugin.video.tehnik/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.tehnik" name="TehniK.ro" version="2.0.7" provider-name="Rujinschi Remus (deltha@gmail.com)">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
+    <import addon="script.module.xbmcswift2" version="2.4.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <language>ro</language>
+    <summary lang="en">IT and Consumer electronics news and trends in Romania</summary>
```

Comment:
for `lang` please use `en_gb` as stated [here](http://kodi.wiki/view/Addon.xml#Required_elements)
Applies to the `description` tag too